.github/workflows/pr_test.yml

```diff
@@ -1,4 +1,4 @@
-on: pull_request
+on: [push, pull_request]
```

Comment:
So currently (due to `pull_request`), once a PR is open, every push to the PR causes the pr_test.yml jobs to be triggered but what happens when `push` is added as well? I presume that a push to a PR will cause two separate jobs to spin up (which would be an unnecessary extra workload).

I typically run the full test suite locally while developing and so I don't use CI to validate my work while I'm developing. Instead, I use the local `make test` (see [`docs/TESTING.md`](https://github.com/fastly/cli/blob/main/TESTING.md)) and when ready to open a PR, this will cause the full test suite to be run again (just to ensure there are no odd things with the CI environment).

Reply:
got it, and no probl.. I missed that doc / instructions of how to run tests locally, but it is fine pointing to them

Reply:
reverted, marking as resolved